spec/csv_importer_spec.rb

```diff
@@ -134,6 +134,44 @@ class ImportUserCSVByFirstName
         "confirmed_at" => Time.new(2012)
       )
     end
+
+    it 'imports even if there are comment lines' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.